cmd/gopogh/main.go

```diff
@@ -15,6 +15,7 @@ import (
 var Build string
 
 var (
+	dbPath         = flag.String("db_path", "", "path to sql summary output (file). Set DB_BACKEND env variable to 'sqlite' for file output")
```

Comment:
if this is an environment variable, does it have to be a flag still ?

Reply:
It doesn't but I thought it could make sense to also have that as an option (triage also has a flag for the env variables it uses)